package med.voll.api.exceptions;

public class UsuarioEncontradoException extends RuntimeException{

}
